docs/examples/go-ipfs-with-experimental/main.go

```diff
@@ -0,0 +1,354 @@
+package main
```

Comment:
IIRC this is mostly the same as the other example. Should we just use a `--experimental` flag in `main.go` to disambiguate?

Is it useful for these to be separate binaries?